Gemfile

```diff
@@ -3,12 +3,18 @@ source "https://rubygems.org"
 
 gemspec
 
-%w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
-  library_path = File.expand_path("../../#{lib}", __FILE__)
+{
+  'rspec' => 'rspec-metagem',
+  'rspec-core' => 'rspec-core',
+  'rspec-expectations' => 'rspec-expectations',
+  'rspec-mocks' => 'rspec-mocks',
+  'rspec-support' => 'rspec-support'
+}.each do |lib, repo|
+  library_path = File.expand_path("../../#{repo}", __FILE__)
   if File.exist?(library_path) && !ENV['USE_GIT_REPOS']
     gem lib, :path => library_path
   else
-    gem lib, :git => "https://github.com/rspec/#{lib}.git", :branch => ENV.fetch('RSPEC_BRANCH', 'main')
+    gem lib, :git => "https://github.com/rspec/#{repo}.git", :branch => ENV.fetch('RSPEC_BRANCH', 'main')
```

Comment:
We do it slightly differently in other repose eg https://github.com/rspec/rspec-expectations/blob/838952dc1416c943e7933684c47249a77481e7f5/Gemfile#L6

Otherwise - thank you!

Reply:
👍🏼 

I’m putting Jon as co-committer since I am basically using his commit (https://github.com/rspec/rspec-expectations/commit/cd9889bf7926844dd2000913f644118a8cc2a54d, https://github.com/rspec/rspec-core/commit/5dfc2077eaa2bc812e68240985616081836b4448, https://github.com/rspec/rspec-mocks/commit/ce1291fd53e2e435684492be3b66d0b9bf884969 etc.)